tests/kernels/gnn_kernels/test_hu_iclr_2020.py

```diff
@@ -0,0 +1,98 @@
+"""
+Tests to check whether the refactored implementation of the code from
+Hu et al. Strategies for Pre-training Graph Neural Networks. ICLR 2020
+(https://github.com/snap-stanford/pretrain-gnns)
+runs and produces node embeddings with the provided pre-trained parameters.
+"""
+
+import os
+
+import pytest
+import torch
+from gprotorch.dataloader import DataLoaderMP
```

Comment:
I think the "optimise imports" command in PyCharm (can't remember which dropdown menu it's under) will re-order these automatically so we have 1. python modules, 2. third-party modules and 3. our own modules in separate blocks in line with PEP8.

Reply:
I formatted them with isort. But should probably run the entire codebase through a reformatter anyway, once we're there.